zdai/api/languageapi.py

```diff
@@ -48,3 +48,21 @@ def get(self, request_id: str) -> Tuple[LanguageClassificationRequest, ApiCall]:
         caller.send()
 
         return LanguageClassificationRequest(api = self, json = caller.response.json()), caller
+
+    def get_multiple(self, request_ids: List[str]) -> Tuple[List[LanguageClassificationRequest], ApiCall]:
+        """
+        Gets multiple Language statuses
+
+        """
+        caller = self._call.new(method='GET', path=f'languages')
+        caller.add_parameter('request_id', request_ids)
+        caller.send()
+
+        language_requests = []
+
+        for request_id, result in caller.response.json().get('statuses').items():
+            r = result
+            r['request_id'] = request_id
+            language_requests.append(LanguageClassificationRequest(api = self, json = r))
```

Comment:
I think you can drop the assignment to `r` here (and the other places that follow the same pattern).

```suggestion
            result['request_id'] = request_id
            language_requests.append(LanguageClassificationRequest(api=self, json=result))
```

Reply:
5f656f1e53c5db6fdba711a10b14bc92a3e4b71e